cfme/containers/provider/__init__.py

```diff
@@ -142,21 +162,23 @@ class ContainersProvider(BaseProvider, Pretty):
     quad_name = None
     db_types = ["ContainerManager"]
     endpoints_form = ContainersProviderEndpointsForm
-    all_view = ContainersProvidersView
-    details_view = ContainersProviderDetailsView
+    all_view = ContainerProvidersView
```

Comment:
Hopefully everything trying to use these views was doing so though these attributes